text_extensions_for_pandas/array/tensor.py

```diff
@@ -35,6 +35,7 @@ class TensorDtype(pd.api.extensions.ExtensionDtype):
     """
     Pandas data type for a column of tensors with the same shape.
     """
+    base = None
```

Comment:
What does this do? Should this change be repeated on the dtypes for span arrays?

Reply:
I'm not sure, I can't find any documentation for it. It's in some extension types and only set to something for a couple. Without it, there is an error in a cython function when using integer as index. I'll followup with a question to the mailing list and try to add document or a fix upstream.